src/platforms/python/guides/fastapi/index.mdx

```diff
@@ -86,6 +86,37 @@ The parameter `traces_sample_rate` needs to be set when initializing the Sentry
 
 </Note>
 
+## Data collected
+
+Here an overview of all the data that is collected by the FastAPI integration
```

Comment:
```suggestion
Here's an overview of all the data that's collected by the FastAPI integration:
```

Reply:
not there anymore when we have two tables.